src/borg/helpers.py

```diff
@@ -528,6 +528,17 @@ def _match(self, path):
         raise NotImplementedError
 
 
+class PathFullPattern(PatternBase):
+    """Full match of a path."""
+    PREFIX = "pf"
```

Comment:
naming:

we have PathPrefixPattern pp:, so PathFullPattern pf: was chosen for consistency.